### ISSUE TITLE
Saving an image in parallel with scalable rendering and screen capture from a session file has missing plots.

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1039]
If you run the script at the end of this description in the test directory the image saved will have legends but not plots. Turning off screen capture fixes the problem. Once this bug is fixed then the changes made to "tests/session/legendproperties" to turn off screen capture should be reversed and the results from that test should be removed from the skip list.

Here is the script:

# To run the test, cd to trunk/test and execute:
# ../src/bin/visit -cli -nowin -geometry "312x328+32+32" -s test.py

ra = GetRenderingAttributes()
ra.scalableActivationMode = ra.Always
SetRenderingAttributes(ra)

OpenComputeEngine("localhost", ("-np", "2"))

RestoreSession("tests/session/legendproperties.session", 0)
sa = SaveWindowAttributes()
sa.screenCapture = 1
sa.width,sa.height = 300,300
sa.family   = 0
sa.fileName = "abc00"
sa.format   = sa.PNG
SetSaveWindowAttributes(sa)
SaveWindow()

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 08/09/2010 07:50 pm
Original update: 08/17/2010 04:06 pm
Ticket number: 337
